lib/thumbs/pull_request_worker.rb

```diff
@@ -1,47 +1,66 @@
 module Thumbs
   class PullRequestWorker
     attr_reader :build_dir
-    attr_reader :build_status
-    attr_accessor :build_steps
-    attr_reader :minimum_reviewers
     attr_reader :repo
     attr_reader :pr
-    attr_accessor :thumb_config
     attr_reader :log
     attr_reader :org
-    attr_accessor :client
-    attr_reader :pull_request_id
+    attr_reader :thumbs_config_default
 
     def initialize(options)
-      @repo = options[:repo]
-      @org = repo.split('/').first
-      @client = Octokit::Client.new(:netrc => true)
-      @pr = @client.pull_request(options[:repo], options[:pr])
-      @pull_request_id=get_pull_request_id
-      @build_dir=options[:build_dir] || "/tmp/thumbs/#{build_guid}"
-      persisted_build_status = read_build_status
-      @build_status = persisted_build_status || {:steps => {}}
-      @build_steps = []
+      if options[:url]
+        @repo, @pr = parse_github_pr_url(options[:url])
+      end
+      @repo ||= options[:repo]
+      @org = @repo.split('/').first
+      @pr ||= options[:pr]
```

Comment:
The previous meaning of `@pr` was not symmetric w/ the `options[:pr]`. Generally, the attribute and initialization options symbol should match. To this end, I changed what was `@pr` to be `pull_request`.